CONTRIBUTING.md

```diff
@@ -0,0 +1,291 @@
+
+#### Before Submitting an Enhancement
+
+- Make sure that you are using the latest version.
+- Read the [documentation](https://www.designsystemet.no/) carefully and find out if the functionality is already covered, maybe by an individual configuration.
+- Perform a [search](https://github.com/digdir/designsystem/issues) to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.
+- Find out whether your idea fits with the scope and aims of the project. It's up to you to make a strong case to convince the project's developers of the merits of this feature. Keep in mind that we want features that will be useful to the majority of our users and not just a small subset. If you're just targeting a minority of users, consider writing an add-on/plugin library.
+
+#### How Do I Submit a Good Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://github.com/digdir/designsystem/issues).
+
+- Use a **clear and descriptive title** for the issue to identify the suggestion.
+- Provide a **step-by-step description of the suggested enhancement** in as many details as possible.
+- **Describe the current behavior** and **explain which behavior you expected to see instead** and why. At this point you can also tell which alternatives do not work for you.
+- You may want to **include screenshots and animated GIFs** which help you demonstrate the steps or point out the part which the suggestion is related to. You can use [this tool](https://www.cockos.com/licecap/) to record GIFs on macOS and Windows, and [this tool](https://github.com/colinkeenan/silentcast) or [this tool](https://github.com/GNOME/byzanz) on Linux. <!-- this should only be included if the project has a GUI -->
+- **Explain why this enhancement would be useful** to most Designsystemet users. You may also want to point out the other projects that solved it better and which could serve as inspiration.
+
+### Improving The Documentation
+
+If you have suggestions on improving the documention either on [www.designsystemet.no](https://www.designsystemet.no/) or in the Github repo, please feel free to create an [Issue](https://github.com/digdir/designsystem/issues/new/choose) with the `Feature Request` template. All contributions are very much appriciated!
+
+### Getting involved with development
+
+It's fantastic that you want to join in and help with our development efforts!
+We have established two contribution levels to suit task size: the first level for smaller tasks and the second for larger ones. The main difference is how involved you will be with the Design System Team.
+
+Unsure which level to choose? Send us an email at <designsystem@digdir.no> and we will get back to you as soon as we can!
```

Comment:
Maybe add just that they can contact us on slack also here?

Reply:
Added slack contact